HealthcareSystem/UserServiceTests/IntegrationTests/UserServiceTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Backend.Model;
+using Microsoft.AspNetCore.Mvc;
+using UserService.CustomException;
+using UserService.Model;
+using UserService.Repository;
+using UserService.Repository.Implementation;
+using UserService.Service.Implementation;
+using Xunit;
+
+namespace UserServiceTests.IntegrationTests
+{
+    public class UserServiceTests
+    {
+        private UserService.Service.Implementation.UserService SetupRepositoryAndService()
+        {
+            DbContextInMemory testData = new DbContextInMemory();
+            testData.SeedDatas();
+            MyDbContext context = testData._context;
+            var userRepo = new UserRepository(context);
+            return new UserService.Service.Implementation.UserService(userRepo);
+        }
```

Comment:
I feel like returning IUserService here would've been better, just to make the tests a bit less rigid?